test/integration/use_int_test.go

```diff
@@ -121,13 +122,28 @@ func (suite *UseIntegrationTestSuite) TestReset() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	cp := ts.SpawnWithOpts(e2e.WithArgs("checkout", "ActiveState-CLI/Python3"))
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("checkout", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("SHELL=bash"),
+	)
 	cp.Expect("Skipping runtime setup")
 	cp.Expect("Checked out project")
 	cp.ExpectExitCode(0)
 
+	// Create a zsh RC file to ensure that the path is updated for both bash and zsh.
+	var zshRcFile string
+	var err error
+	if runtime.GOOS != "windows" {
+		zsh := &zsh.SubShell{}
+		zshRcFile, err = zsh.RcFile()
+		suite.NoError(err)
+		err = fileutils.TouchFileUnlessExists(zshRcFile)
+		suite.NoError(err)
+	}
+
 	cp = ts.SpawnWithOpts(
 		e2e.WithArgs("use", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("SHELL=bash"),
```

Comment:
Only the test with the zsh assertion needs this.

Reply:
I believe that setting `SHELL` here ensures that `bash` is the active shell so that we can tests that the `.zshrc` is also updated.